src/navigation/screenOptions.tsx

```diff
@@ -49,11 +53,13 @@ export const getScreenOptions =
           {withDrawer && <DrawerHeader navigation={navigation} style={styles.icon} />}
         </WrapperRow>
       ),
-      headerLeft: withFavorites
-        ? () => <FavoritesHeader navigation={navigation} style={styles.icon} />
-        : HeaderLeft,
+      headerLeft:
+        !noHeaderLeft &&
+        (withFavorites
+          ? () => <FavoritesHeader navigation={navigation} style={styles.icon} />
+          : HeaderLeft),
       title: route.params?.title ?? '',
-      cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS
+      cardStyleInterpolator: cardStyleInterpolator ?? CardStyleInterpolators.forHorizontalIOS
```

Comment:
What are the differences between this and `forScaleFromCenterAndroid` used in `defaultStackConfig`?

Reply:
https://reactnavigation.org/docs/5.x/stack-navigator#cardstyleinterpolators, sadly `forScaleFromCenterAndroid` is not listed and explained there. but a new screen is not coming from the right but is scaling from the center as the name suggests and you might already checked out.

Reply:
Thanks 👍